docs/sql-ref-syntax-aux-describe-query.md

```diff
@@ -19,4 +19,34 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The DESCRIBE QUERY statement is used to return the metadata of output
+of a query. 
+
+### Syntax
+{% highlight sql %}
+{DESC | DESCRIBE} [QUERY] query 
+
+query:= {cte | select-query | inline-table | TABLE table-name}
+{% endhighlight %}
+**Note**
+* For detailed syntax of query parameter please refer to [select-statement](sql-ref-syntax-qry-select.html)
+  and the examples below.
+* DESC and DESCRIBE are interchangeble and mean the same thing.
+
+### Examples
+{% highlight sql %}
+-- Returns column name, column type and comment info for a simple select query
+DESCRIBE QUERY select * customer;
+-- Returns column name, column type and comment info for a inline table.
+DESC QUERY VALUES(1.00D, 'hello') as test_tab(c1, c2);
```

Comment:
I don't know SQL well enough to know: you can write 1.00D as a literal? the "D" works?

Reply:
@srowen Yeah :-). Here is the grammar snippet for a double literal.
```
DOUBLE_LITERAL
    : DIGIT+ EXPONENT? 'D'
    | DECIMAL_DIGITS EXPONENT? 'D' {isValidDecimal()}?
```